stac-asset-service/notebooks/pystac-stac-asset-service.ipynb

```diff
@@ -0,0 +1,1667 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Access UP42 assets with PySTAC\n",
+    "\n",
+    "[UP42](https://up42.com/) is a platform for accessing data from various geospatial providers.\n",
+    "\n",
+    "* [SpatioTemporal Asset Catalog](https://stacspec.org/en/) (STAC) is a specification designed to establish a standard in geospatial data.\n",
+    "* [PySTAC](https://pystac.readthedocs.io/en/stable/) is a library for working with STAC.\n",
+    "\n",
+    "This tutorial explains how to get STAC collections and STAC items associated with the UP42 assets in your storage and visualize them on a map. For more information, see [UP42's introduction to STAC](https://docs.up42.com/developers/api-assets/stac-about).\n",
+    "\n",
+    "1. [Import libraries](#import)\n",
+    "2. [Create an UP42 access token](#token)\n",
+    "3. [Create a PySTAC-Client connection](#pystac-client)\n",
+    "4. [Get STAC collections](#collections)\n",
+    "5. [Get STAC items](#items)\n",
+    "6. [Make a detailed STAC item search](#search)\n",
+    "7. [Download and visualize results](#download-visualize)\n",
+    "\n",
+    "## Import libraries <a class=\"anchor\" id=\"import\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import up42\n",
```

Comment:
If the SDK also makes available `pystac_client` why do you do things as if the SDK wasn't available? E.g., the auth, the downloading of assets, etc.